examples/timer-log-cdi/src/main/resources/application.properties

```diff
@@ -14,33 +14,17 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
+
 #
 # Quarkus
 #
 quarkus.banner.enabled = false
 
-#
-# Quarkus - Camel
-#
-
-# disable build time route discovery as the only
-# route defined in this project is based on CDI.
-#
-# This is not strictly needed for the correctness
-# because camel-quarkus automatically
-# filters auto-discovered routes that targets CDI
-# but if is is known that routes are all leveraging
-# CDI, this option will give some little boost to
-# the build process
-quarkus.camel.main.routes-discovery.enabled = false
-
-#
-# Camel
-#
-camel.context.name = quarkus-camel-example-timer-log-cdi
+# camel look-up beans using BeanManager so we don't want
+# ArC to remove beans without injection points.
+quarkus.arc.remove-unused-beans = false
```

Comment:
Revisit this if this becomes a common use - I would like end users to not know or have to use such an option.

Reply:
It is possible to use the `@Unremovable` annotation as alternative but it's not better.

We probably need to discuss this topic with the Quarkus team and see if through a build item it could be possible to reverse the behaviour so not to remove beans unless that feature is turned on explicitly.

Reply:
I think we should create a ticket about this, so its not forgotten. As end users this is a PITA to remember to have to do

Reply:
https://github.com/apache/camel-quarkus/issues/1387